Source/WebCore/rendering/updating/RenderTreeBuilderMultiColumn.cpp

```diff
@@ -188,10 +188,11 @@ void RenderTreeBuilder::MultiColumn::restoreColumnSpannersForContainer(const Ren
     auto& spanners = multiColumnFlow.spannerMap();
     Vector<RenderMultiColumnSpannerPlaceholder*> placeholdersToRestore;
     for (auto& spannerAndPlaceholder : spanners) {
-        auto& placeholder = *spannerAndPlaceholder.value;
-        if (!placeholder.isDescendantOf(&container))
-            continue;
-        placeholdersToRestore.append(&placeholder);
+        if (auto* placeholder = spannerAndPlaceholder.value.get()) {
```

Comment:
can we at least ASSERT(placeholder)?

Reply:
I think adding an ASSERT here will make debug builds fail, I'll try debugging the true cause then.